src/main/java/org/topbraid/jenax/util/DelegatingDataset.java

```diff
@@ -200,4 +200,40 @@ public TxnType transactionType() {
     public ReadWrite transactionMode() {
         return delegate.transactionMode();
     }
+
+
+	@Override
+	public Model getNamedModel(Resource uri) {
+		return delegate.getNamedModel(uri);
+	}
+
+
+	@Override
+	public boolean containsNamedModel(Resource uri) {
+		return delegate.containsNamedModel(uri);
+	}
+
+
+	@Override
+	public Dataset addNamedModel(Resource resource, Model model) {
+		return delegate.addNamedModel(resource, model);
+	}
+
+
+	@Override
+	public Dataset removeNamedModel(Resource resource) {
+		return delegate.removeNamedModel(resource);
+	}
+
+
+	@Override
+	public Dataset replaceNamedModel(Resource resource, Model model) {
+		return delegate.replaceNamedModel(resource, model);
+	}
```

Comment:
The methods above (`addNamedModel`, `removeNamedModel`, and `replaceNamedModel`) should return `this`, not what is returned by the `delegate`'s implementations. I'm guessing this would not usually be a problem, but it does violate the Javadoc contract.